lib/factory_trace/helpers/caller.rb

```diff
@@ -1,11 +1,13 @@
+# frozen_string_literal: true
+
 module FactoryTrace
   module Helpers
     module Caller
       module_function
 
       # @return [String] file and line where the original method was called
       def location
-        location = caller_locations[1]
+        location = caller_locations(2..2).first
```

Comment:
**Question:** Why do we need this change?

Reply:
It requests only the single stack frame desired instead of requesting the full stack trace and throwing away all but one element. Thus much faster.

https://github.com/rubocop/rubocop-performance/commit/b78575b9f0c3018406bbc6e4904a24260e9ce3bd